SeniorInterview/StocksAnalyzed/src/Main.java

```diff
@@ -8,8 +9,8 @@ public static void main(String[] args) {
 
         // Variacao absoluta
```

Comment:
I think It'd be nice to have the comment in english, this way the code can be easyly understood by any developer around the world. Besides that, could you please add more details to the comment? Absolute variation of what e.g.